src/app/shared/filesystem/file-list-view/file-list-view.component.scss

```diff
@@ -33,12 +33,12 @@
 
 .grid-row {
   display: grid;
-  grid-template-columns: 2fr 2fr 1fr 25px;
+  grid-template-columns: 2fr 2fr 2fr 1fr 25px;
```

Comment:
The file size column shouldn't need 2fr width, should probably be 1fr so that it shrinks before the name and description fields

Reply:
Yup, I agree. 1fr is fine. I'll change it.